src/frontend/components/SøknadsSteg/OmBarnet/AndreForelder.tsx

```diff
@@ -152,35 +177,110 @@ const AndreForelder: React.FC<{
                                     inkluderVetIkke={true}
                                     språkValues={{ navn: barnetsNavn }}
                                 />
-                                <LandDropdown
-                                    felt={skjema.felter.andreForelderArbeidUtlandetHvilketLand}
-                                    skjema={skjema}
-                                    dynamisk
-                                    label={
-                                        <SpråkTekst
-                                            id={
-                                                omBarnetSpørsmålSpråkId[
-                                                    andreForelder[
-                                                        andreForelderDataKeySpørsmål
-                                                            .arbeidUtlandetHvilketLand
-                                                    ].id
-                                                ]
+                                {toggles.EØS_KOMPLETT ? (
```

Comment:
Kan vi gjenbruke `<Arbeidsperiode>` komponenten her? eller er de for forskjellig?